instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -23,7 +23,7 @@ class PostsPage extends Component {
       return { 
```

Comment:
glad you were able to put in both the no post component and the dynamic filtering, great work